### ISSUE TITLE
Regression of #18097: Cloud Keys search is not working correctly

### ISSUE THREAD

**Issue by davidnuzik** [rancher/rancher#18890]
Version: Master 2.2 (3/13/19)

**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
- Have several cloud keys
- Search for something that is not among any of the keys (name, date, etc)
- Also try the letter g and results come up that don't have the letter g

**Result:**
You will still get rsults when searching for something that is not seen on the page listing the Cloud Credentials. It looks like one example is it is searching the long date as seen in the API.

It looks like search is looking at Provider type and at the time as well as the Name.

There is a possibility #18889 is a symptom of this issue.

**Comment by westlywright** [comment]
https://github.com/rancher/ui/pull/2762 should fix this

**Comment by davidnuzik** [comment]
To include in testing: searching just 'a' and 'z' should work as expected now. That issue is the same as this. See #18889 for details.

**Comment by davidnuzik** [comment]
Version: Master 2.2 (3/14/19) 6875782b6131
Confirmed issue. This is now working as expected.
I can also search the letter 'a' and 'z' due to the fixes and not get unexpected results. I also tried searching various numbers and confirmed that we are only searching the group name and cloud credential name.

For example, I can search for "Amazon" and I get all entries under the Amazon group. I think this is good. When you search for a group you get all listings. Below is a search for "amazon"
![image](https://user-images.githubusercontent.com/45179589/54379482-b895f080-4646-11e9-8c73-6791da07f566.png)

This works for other groups as well such as azure, vmware vsphere, and digitalocean. I can explicitly search for names of cloud credentials without any issues. I tried various sets of numbers/alpha names/etc and did not identify any issues. I tried searching for things seen in the API but not the UI and only can search what I see in the UI (Name and group name)
